pipeline/lib/current_bench_json.ml

```diff
@@ -278,16 +294,17 @@ end
 
 module Latest = V2
 
-let version = 2
-let of_json json = Latest.of_json json
-let to_json t = Latest.to_json t
-
 let of_list jsons =
-  List.fold_left (fun acc json -> Latest.merge acc [ of_json json ]) [] jsons
+  List.fold_left
+    (fun acc json ->
+      match Latest.of_json json with
+      | t -> Latest.merge acc [ t ]
+      | exception _ -> acc)
+    [] jsons
```

Comment:
Perhaps we could do a `partition_map` like https://github.com/ocurrent/current-bench/blob/main/pipeline/lib/json_stream.ml#L263 to save the errors? :)
For context, the issue is that some logs have both invalid jsons + valid jsons, but the "invalid" jsons are just not intended for current-bench and so it's fine to ignore them:

https://github.com/ocurrent/current-bench/blob/main/pipeline/lib/current_bench_json.ml#L266

I haven't thought too much about it, but the absence of a toplevel `"results"` field could be used as a hint that the json wasn't for us? :)

Reply:
> save the errors
Something like this?
```ocaml
List.fold_left
 (fun (jsons, exns) (json, range) ->
   try
     ( json
       |> Yojson.Safe.from_string
       |> json_merge_lines range
       |> Current_bench_json.Latest.of_json
       |> Current_bench_json.Latest.add jsons,
       exns )
   with exn ->
     (jsons, (json, range, Printexc.to_string exn) :: exns))
 ([], [])
```